config/android/BUILD.gn

```diff
@@ -55,14 +55,18 @@ config("compiler") {
   if (is_clang) {
     if (current_cpu == "arm") {
       abi_target = "arm-linux-androideabi"
+      cflags += [ "-D__ANDROID_API__=16" ]
```

Comment:
after the for the Unified Headers changes, we need set -D__ANDROID_API__ parameter